packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -94,6 +95,11 @@ export default function Classifier({
   })
 
   const [loaded, setLoaded] = useState(false)
+  const { data } = useSWR(`/workflows/${workflowID}`, client.panoptes.get)
+  let workflowData
+  if (data?.text) {
+    workflowData = data.text
```

Comment:
`workflowData` here includes the full workflow resource, including `workflow.classifications_count` and `workflow.completeness`. I think that should be ok, as they aren't stored in the MobX data model, but it's worth double-checking that changes to the workflow counters don't trigger a full refresh of the classifier, via `useEffect`.

Reply:
eg. someone else classifying, elsewhere in the world, and updating the total classification count shouldn't cause the workflow to reload in my browser and reset my work in progress.